Tests/Command/ActivateUserCommandTest.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+/*
+ * This file is part of the FOSUserBundle package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\UserBundle\Tests\Command;
+
+use FOS\UserBundle\Command\ActivateUserCommand;
+use Symfony\Component\Console\Application;
+use Symfony\Component\Console\Tester\CommandTester;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+class ActivateUserCommandTest extends \PHPUnit_Framework_TestCase
+{
+    public function testExecute()
+    {
+        $commandTester = $this->createCommandTester($this->getContainer('user'));
+        $exitCode = $commandTester->execute(array(
+            'command' => 'fos:user:activate', // BC for SF <2.4 see https://github.com/symfony/symfony/pull/8626
+            'username' => 'user',
+        ), array(
+            'decorated' => false,
+            'interactive' => false,
+        ));
+
+        $this->assertEquals(0, $exitCode, 'Returns 0 in case of success');
+        $this->assertRegExp('/User "user" has been activated/', $commandTester->getDisplay());
+    }
+
+    /**
+     * @group legacy
```

Comment:
this is not really needed, as it is not executed against deprecated APIs (it never runs when the new APIs are there)